### DISCUSSION TITLE
Is anybody using Gemini on Google Home ?

### DISCUSSION THREAD

**Discussion by Ryanenk** [community/community#144744] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

[Gemini is on Google Home](https://googlehomecentral.com/how-to-use-google-home-with-gemini/) now .I have all my smart home devices on Google Home - i do usually use the google assistant (the default voice ) should i opt for Gemini ? Upgrading it is fine or should i Just stay happy with Google assistant , i really wanna try out the AI Feature , But not sure if it makes any huge difference . 
Have any of you tried it ?

**Comment by shinybrightstar** [comment]
Hi @Ryanenk,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
